src/models/ITestResult.ts

```diff
@@ -105,7 +105,7 @@ export interface Defect {
   prohibitionIssued?: boolean;
 }
 
-export interface DefectIVA {
+export interface DefectRequiredStandard {
```

Comment:
Can this just be called RequiredStandard now and remove the Defect part of the name

Reply:
Done, and pushed.